.github/workflows/release_crates.yml

```diff
@@ -0,0 +1,21 @@
+name: Release crates
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 2-6'
+  push:
+    branches:
+      - main
+
+jobs:
+  environment: carte-publish
+  release:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Login to crate.io
+        run: cargo login ${{ secrets.CRATES_IO_TOKEN }}
```

Comment:
This secrets needed to be added. I don't have the permission to add, so I would like to have it added.